### ISSUE TITLE
Couldn't make it work.

### ISSUE THREAD

**Issue by fclaussen** [DanBeckett/acf-taxonomy-relationship#3]
Hi
I tried adding this to a project but it didn't work.
I created the field and assigned it to show for a category.
I can see other fields there but yours will only show the label, not the actual field.

Is this still maintained?

**Comment by DanBeckett** [comment]
Hi, this was last maintained quite a long time ago now. I intend to bring it fully up-to-date but I've unfortunately not been touching much WordPress stuff at the moment so hard to put a time on it, sorry!

**Comment by CHEWX** [comment]
Yep, to confirm, still not working. Would be great to get this going again.

**Comment by sebastian-works** [comment]
Would be great to see an update on this!
